monocle/static/js/main.js

```diff
@@ -357,8 +357,20 @@ function getWorkers() {
     });
 }
 
-var map = L.map('main-map', {preferCanvas: true}).setView(_MapCoords, 13);
+var params = {};
+window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function(m, key, value) {
```

Comment:
This should be handled now with parseFloating the params, thanks for pointing this out. I hadn't thought of clicking the settings and then refreshing/reloading.